.golangci.yml

```diff
@@ -19,6 +19,10 @@ linters:
     - revive
     - gofumpt
     - depguard
+    - nakedret
+    - unconvert
+    - wastedassign
```

Comment:
The only thing I don't like too much is this linter.
That's at least the only one that seems to be related to the following, even though I don't find anything in their docs for it:
Why is it better to use i.e. `var test string` instead of `test := ""`?
Is there any performance implication to it?
Because otherwise, I think `test := ""` is more readable as I then don't have to do the mental gymnastics of what the value is initally...

Reply:
You understand it wrong, it's there because you don't use the initial value, you don't need the mental gymnastics because the value just isn't used :)

It's going to complain if you do `a:=""` but then never use the contents of a.
It's not just empty strings, also for stuff like

````
a := "my value"
a = afunc()
````

Reply:
Ah, I see.
I already wondered why the changes are that limited...